src/main/java/com/mycompany/exercisethree/KidsCalculator.java

```diff
@@ -0,0 +1,72 @@
+package com.mycompany.exercisethree;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+/**
+ *
+ * @author Cristian Gallego
+ */
+public class KidsCalculator extends Calculator {
+
+    @Override
+    public void operateCalculator() {
+
+        Scanner getInt = new Scanner(System.in);
+        System.out.println("WELCOME TO THE KIDS CALCULATOR, PLEASE CHOOSE WHAT YOU WANT TO DO ");
+        System.out.println("1. ADD");
+        System.out.println("2. SUBSTRACT");
+        System.out.println("3. MULTIPLY");
+        System.out.println("4. DIVIDE");
+        System.out.println("5. RETURN");
+        try {
+            option = getInt.nextInt();
+            listNumbers = (option >= 1 && option <= 4) ? getInputString() : null;
+        } catch (InputMismatchException e) {
+            System.out.println("Invalid option, please enter a valid option");
+            getInt.nextLine(); // clear the scanner buffer
+            operateCalculator();
+        }
+
+        switch (option) {
+            case 1 -> {
+                if (listNumbers.size() < 4) {
```

Comment:
You can validate the list before entering to switch instead of adding the if in every option

Reply:
Done